configs/sites/tier1/acorn/packages.yaml

```diff
@@ -41,3 +41,6 @@
       externals:
       - spec: grep@3.1
         prefix: /usr
+    esmf:
+      require:
+      - '+python'
```

Comment:
I hopefully remember this!

Reply:
And we need to check if that works all the time. We have environments (`neptune-env`) that don't require it (yet).